src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentMaintenanceServiceImpl.java

```diff
@@ -125,9 +130,58 @@ public boolean cancelDisbursement(Integer paymentGroupId, Integer paymentDetailI
         }
         return true;
     }
+    
+    private static boolean isCheckAchDisbursement(final PaymentGroup paymentGroup) {
```

Comment:
For the two methods that were explicitly duplicated here due to the superclass declaring them as private, should comments be added to clarify that?

Reply:
added